frontend/src/styles/globals.scss

```diff
@@ -23,7 +23,6 @@ a {
   margin: 0;
 }
 
-html,
```

Comment:
I found that the `100%` height on the `html` element was causing a weird keyboard bug as seen here, where after I close the soft keyboard on iOS it wipes out the back of it due to the changed height that isn't rerendered.

https://user-images.githubusercontent.com/13066134/211921434-4d9b7d97-06b2-49ba-90f2-807a9f63c0a6.mov

Based on the documentation within this section, I think that was only applied to prevent whitespace from appearing in the body size is smaller than the document size.